ansible-roles/os/ubuntu/clevis-install.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
@DaniilKl I force pushed base branch (`encrypted-rootfs-release`), please pull and rebase on top of that. Also please add ansible playbook by following the examples that are already there, which is `ansible-roles/roles/${suite_test_name}/tasks/common.yml`. This is how ansible KW runs those yml files, it could not be executed from the path that you propose here.

Reply:
`clevis` playbook has been deleted due to the fact that we need `clevis` only on Ubuntu with encrypted `rootfs`. `clevis` is being installed now only in test suite.